pkg/controller/pkg/revision/reconciler.go

```diff
@@ -224,12 +250,12 @@ func SetupConfigurationRevision(mgr ctrl.Manager, l logging.Logger, namespace st
 	}
 
 	r := NewReconciler(mgr,
-		clientset,
+		cache,
 		WithHooks(NewConfigurationHooks()),
 		WithNewPackageRevisionFn(nr),
 		WithParser(parser.New(metaScheme, objScheme)),
-		WithParserBackend(parser.NewPodLogBackend(parser.PodNamespace(namespace))),
-		WithLinter(NewPackageLinter([]PackageLinterFn{OneMeta}, []ObjectLinterFn{IsConfiguration}, []ObjectLinterFn{Or(IsXRD, IsComposition)})),
+		WithParserBackend(NewImageBackend(Client(clientset))),
```

Comment:
Should this have a cache, etc?

Reply:
Yep :+1: was just testing one at a time and forgot to update